packages/fiori/src/Wizard.ts

```diff
@@ -864,7 +887,7 @@ class Wizard extends UI5Element {
 
 			return {
 				icon: step.icon,
-				titleText: step.titleText,
+				titleText: step.titleText ? step.titleText : "",
```

Comment:
You're correct. Since WizardStep.titleText is string property with default value "" this seems redundant. Also this object for me should be described as a type.